bitrise.yml

```diff
@@ -365,18 +305,21 @@ workflows:
 
             echo "$BITRISE_ARTIFACT_PATHS"
 
+            derived_data=${BITRISE_ARTIFACT_PATHS#*|}
```

Comment:
Not familiar with the `#*|` but I assume its needed.

Reply:
yes, the reason is that in the BITRISE_ARTIFACT_PATHS there are two .zip files, they appear as:
`BITRISE_ARTIFACT_PATHS=/foo/bar/file1.zip|/foo/bar/file2.zip`
We need to unzip only the second one, the | character is the separator between both. With that command we get what is after the pipe.
Not sure if there is a better way, happy to learn and use it if that's the case.